templates/extensions/the-graph/README.md.args.mjs

```diff
@@ -0,0 +1,303 @@
+export const extraContents = `## 🚀 Setup The Graph Integration
+
+Now that we have spun up our blockchain, started our frontend application and deployed our smart contract, we can start setting up our subgraph and utilize The Graph!
+
+> Before following these steps be sure Docker is running!
+
+&nbsp;
+
+#### ✅ Step 1: Clean up any old data and spin up our docker containers ✅
+
+First run the following to clean up any old data. Do this if you need to reset everything.
+
+\`\`\`
+yarn clean-node
+\`\`\`
+
+> We can now spin up a graph node by running the following command… 🧑‍🚀
+
+\`\`\`
+yarn run-node
+\`\`\`
+
+This will spin up all the containers for The Graph using docker-compose. You will want to keep this window open at all times so that you can see log output from Docker.
+
+> As stated before, be sure to keep this window open so that you can see any log output from Docker. 🔎
+
+> NOTE FOR LINUX USERS: If you are running Linux you will need some additional changes to the project.
+
+##### Linux Only
+
+**For hardhat**
+
+Update your package.json in packages/hardhat with the following command line option for the hardhat chain.
+
+\`\`\`
+"chain": "hardhat node --network hardhat --no-deploy --hostname 0.0.0.0"
+\`\`\`
+
+**For foundry**
+
+Update your package.json in packages/foundry with the following command line option for the anvil chain.
+
+\`\`\`
+"chain": "anvil --host 0.0.0.0 --config-out localhost.json",
+\`\`\`
+
+Save the file and then restart your chain in its original window.
+
+\`\`\`
+yarn chain
+\`\`\`
+
+Redeploy your smart contracts.
+
+\`\`\`
+yarn deploy
+\`\`\`
+
+You might also need to add a firewall exception for port 8432. As an example for Ubuntu... run the following command.
+
+\`\`\`
+sudo ufw allow 8545/tcp
+\`\`\`
+
+&nbsp;
+
+#### ✅ Step 2: Create and ship our Subgraph ✅
+
+Now we can open up a fifth window to finish setting up The Graph. 😅 In this fifth window we will create our local subgraph!
+
+> Note: You will only need to do this once.
+
+\`\`\`
+yarn local-create
+\`\`\`
+
+> You should see some output stating your Subgraph has been created along with a log output on your graph-node inside docker.
```

Comment:
"Subgraph" is usually lower-cased

Reply:
Updated at [53c1542](https://github.com/scaffold-eth/create-eth/pull/25/commits/53c15426b6df32e6fd38a52e8247f01011c26980)